spec/index.adoc

```diff
@@ -3994,27 +3994,6 @@ S with
     canister_state_counter[CanisterId] = N1
 ....
 
-==== Trimming canister history
```

Comment:
I think it would still make sense to trim the history entries in the replicated state at 1k entries to avoid unbounded growth of the state, no?

Reply:
You're right. Let me add this back.